src/index.md

```diff
@@ -40,15 +40,17 @@ The ecosystem consist of various components:
 
 ## Manage Account & Project
 - [Permissions](./manage/permissions/)
-- [Synchronisation](./manage/synchronisation/)
+- [Behind Data Synchronisation](./manage/synchronisation/)
 - [How to Create a New Project](./manage/create-project/)
-- [Home Page and Dashboard](./manage/dashboard/)
 - [How to Share, Transfer or Delete a Project](./manage/project-advanced/)
+- [How to Deploy Revised Projects](./manage/deploy-new-project/)
+- [How to Recover Missing Data](./manage/missing-data/)
 - [How to Use Mergin Maps Plugin for QGIS with Multiple Servers](./manage/plugin-multi-server-use/)
-- [How to Synchronise Project](./manage/plugin-sync-project/)
+- [Download and Synchronise projects with QGIS Plugin](./manage/plugin-sync-project/)
+- [Home Page and Dashboard](./manage/dashboard/)
 - [Project History and Versions](./manage/project-details/)
 - [Selective Synchronisation](./manage/selective_sync/)
-- [Data Synchronisation and Conflicts](./manage/data_sync/)
+- [Data Synchronisation](./manage/data_sync/)
```

Comment:
I would rename/rewrite it to "How to synchronise data" and maybe even merge it to "./manage/plugin-sync-project/"?

or alternatively move "Manual data transfer (android/iOS) to "How to Recover Missing Data", since that is only place where the user should do it and remove the article otherwise

Reply:
There is a comment "IMPORTANT: This permalink is referenced from Input App" so maybe it would be safe to keep it and changing the title to "how to synchronise data" ?

Reply:
we have redirects , so it is ok to rename anythiing